jaeger_client/tracer.py

```diff
@@ -237,8 +239,8 @@ def report_span(self, span):
         self.reporter.report_span(span)
         self.metrics.spans_finished(1)
 
-    def random_id(self):
-        return self.random.getrandbits(constants.MAX_ID_BITS)
+    def random_id(self, max_id_bits=constants.MAX_ID_BITS):
```

Comment:
This constant is no longer adequate/ misleading. Not sure why it was public in the first place either. I think we should label it deprecated and define separate constants for max bits for trace and span ids, preferably private. 

The method should not have a default.

Reply:
Something like `_max_trace_id_bits` and `_max_span_id_bits`/`_max_id_bits` in `constants.py`?

Reply:
yes.